app-emulation/wine-vanilla/metadata.xml

```diff
@@ -18,6 +18,10 @@ This variant of the Wine packaging does not include external patchsets
 	</longdescription>
 	<use>
 		<flag name="capi">Enable ISDN support via CAPI</flag>
+		<flag name="crossdev-mingw">
+			Use <pkg>sys-devel/crossdev</pkg> for the toolchain rather than
```

Comment:
You probably know more about people using mingw on gentoo than me, but would anyone even want to do it the old way? If not, we can consider only using mingw-toolchain

Reply:
Unsure, I see people do all kind of strange things with this (even use llvm-mingw) so thought to keep it, but ultimately they could put mingw64-toolchain in package.provided and call it a day -- then we can drop all the pkg_pretend stuff and USE.

If you want it this way too, then I'm all for it.

Reply:
I guess lets keep it for now, i don't think its causing any harm

Reply:
Alright, I guess no changes needed anywhere so far then, keeping as-is :)

Reply:
Let me test this locally quickly and then we can push and sync to ::gentoo whenever youre ready

Reply:
It's ready anytime as far as I'm concern, unless you spot something I missed.